File: 0-1000 C/QUALIFY.c
#include <stdio.h>
int main(){
    int T, X, A, B;
    scanf("%d", &T);
    for (int i = 0; i < T; i++){
        scanf("%d %d %d", &X, &A, &B);
        (A + 2*B >= X) ? printf("Qualify\n") : printf("NotQualify\n");
    }
    return 0;
}

File: 0-1000 Python/QUALIFY.py
T = int(input())

for i in range(T):
    temp = input().split(" ")
    X, A, B = int(temp[0]), int(temp[1]), int(temp[2])

    if A + 2*B >= X:
        print("Qualify")
    else:
        print("NotQualify")
